src/Ocelot/DependencyInjection/ServiceCollectionExtensions.cs

```diff
@@ -65,4 +69,22 @@ public static IOcelotBuilder AddOcelotUsingBuilder(this IServiceCollection servi
     {
         return new OcelotBuilder(services, configuration, customBuilder);
     }
```

Comment:
```c#
configuration ??= services.FindConfiguration(null);
return new OcelotBuilder(services, configuration, customBuilder);
```

Reply:
Fixed in https://github.com/ThreeMammals/Ocelot/pull/1986/commits/293c1d9d5b5bd30b7ebcfadfc50c71b69487118b ✔️